ui/src/views/network/CreateL2NetworkForm.vue

```diff
@@ -114,15 +114,15 @@
             </a-select>
           </a-form-item>
           <a-form-item v-if="!this.isObjectEmpty(this.selectedNetworkOffering) && this.selectedNetworkOffering.specifyvlan">
-            <tooltip-label slot="label" :title="$t('label.vlan')" :tooltip="apiParams.vlan ? apiParams.vlan.description : null"/>
+            <tooltip-label slot="label" :title="$t('label.vlan')" :tooltip="apiParams.vlan?.description || null"/>
```

Comment:
@Damans227  Don't apply simplify code to <template> because it won't work.

Reply:
Ok, I have rolled back these changes @utchoang